include/clang/AST/ASTContext.h

```diff
@@ -2486,12 +2488,16 @@ class ASTContext : public RefCountedBase<ASTContext> {
   /// at least as much checking as the other type T2.  Return true if it does
   /// or false if it does not or the types differ in some other way than
   /// checkedness.
+  /// Note:: Checked C - In bounds safe interface scopes, this function
```

Comment:
You don't need to prefix this with "Checked C", as this function is specific to the Checked C implementation.